libraries/AP_Volz_Protocol/AP_Volz_Protocol.cpp

```diff
@@ -0,0 +1,166 @@
+/*
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <AP_HAL/AP_HAL.h>
+#include "AP_Volz_Protocol.h"
+
+extern const AP_HAL::HAL& hal;
+
+const AP_Param::GroupInfo AP_Volz_Protocol::var_info[] = {
+
+    // @Param: ENABLED
+    // @DisplayName: Enable of volz servo protocol
+    // @Description: outputs volz servo protocol to serial out.
+    // @Values: 0:Disabled,1:Enabled
+    // @User: Standard
+    AP_GROUPINFO("ENABLED", 0, AP_Volz_Protocol, enabled, 0),
```

Comment:
this "enabled" is not necessary, please remove. Setting the serial port does this

Reply:
removed